content/applications/hr/frontdesk.rst

```diff
@@ -0,0 +1,318 @@
+  served.
+- :guilabel:`Last Check-In`: This indicates how long ago the last check-in took place, in hours
+  and/or minutes.
+- :guilabel:`Additional options menu`: Click this to reveal a drop-down menu, to access either the
+  :guilabel:`Kiosk` in a separate tab, the full list of planned and/or checked-in
+  :guilabel:`Visitors` for the current day, the :guilabel:`Statistics` for that station, or the
+  :guilabel:`Configuration` options for that particular station.
+- :guilabel:`Assign`: The user who is notified when someone checks-in at this station. Add another
+  user by clicking the :guilabel:`Assign` icon and selecting any additional users desired.
+
+.. image:: frontdesk/main-lobby.png
+   :align: center
+   :alt: Frontdesk kiosk card with the stats shown on it.
+
+Click on any station card to navigate to a complete list of all visitors associated with that
+particular station, past, present, and future.
+
+.. _frontdesk/visitors:
+
+Visitors
+========
+
+To access a complete list of visitors that are currently checked in, navigate to
+:menuselection:`Frontdesk application --> Visitors`. The visitors are listed in a list view, with
+the following details that were entered when they checked-in:
+
+- :guilabel:`Name`: The guest's name.
+- :guilabel:`Visitor Company`: The guest's company they represent.
+- :guilabel:`Phone`: The guest's phone number.
+- :guilabel:`Drinks`: The drink the guest requested.
+- :guilabel:`Host`: Who the guest is waiting to see.
+- :guilabel:`CheckIn`: The date and time the guest checked-in.
+- :guilabel:`Checkout`: The date and time the guest checked-out. In the default view, only guests
+  with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible. Guests with check-out
+  times are only visible if the filter :guilabel:`Today` is not active.
+- :guilabel:`Duration`: The current amount of time the guest has been checked-in for.
+- :guilabel:`Station`: The location of where the guest checked-in.
+- :guilabel:`Status`: The status of the guest. The options are :guilabel:`Checked-In`,
+  :guilabel:`Planned`, :guilabel:`Checked-Out`, or :guilabel:`Cancelled`. In the default view, only
+  guests with a :guilabel:`Checked-In` or :guilabel:`Planned` status are visible.
+- :guilabel:`Email`: The guest's email address.
+- Status Column: When a guest leaves, click the :guilabel:`Check out` button to update the guest's
+  record and log the date and time they left. If a scheduled guest arrives, if they did not check-in
+  using the *Frontdesk* application kiosk, they can be checked-in on the visitor log. Click
+  :guilabel:`Check in` to log the date and time they arrived.
+- :guilabel:`Drink Served`: When a guest requests a drink, the :guilabel:`Drink Served` button
+  appears at the end of the visitor line. When a drink has been served, click the :guilabel:`Drink
+  Served` button to indicate the drink has been delivered to the guest.
```

Comment:
I agree with the status- I was torn since I was adding 'column', but it looks better bolded with a GUI label. A lot of the info in the different sections overlaps, but I wanted to be thorough, so if anyone jumped to only one section they had a question on, all the info was there.